babel.config.js

```diff
@@ -1,8 +1,4 @@
 module.exports = {
-  plugins: [
-    ['react-native-reanimated/plugin', { // reanimated must be the last package
-      globals: ['__scanCodes'],
```

Comment:
no need for the globals anymore?

Reply:
This was needed for the `vision-camera-code-scanner` setup, which we don't use anymore.